doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -123,60 +120,49 @@ emitters:
+```
 **Note**:
 
-- `<path-to-cadl-file>` - Should be the entrypoint cadl file of the cadl project. e.g. When you define `client.cadl` to customize the client, the `client.cadl` will be the entrypoint cadl file, and you should specify the path to the `client.cadl`.
-  
-#### Build the library project
+- `-service` takes Azure client service directory name. ie. purview. It equals to the name of the directory in the specification folder of the azure-rest-api-specs repo that contains the REST API definition file.
+- For `- namespace`, please use one of the pre-approved namespace groups on the [.NET Azure SDK Guidelines Approved Namespaces list](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespaces-approved-list). This value will also provide the name for the shipped package, and should be of the form `Azure.<group>.<service>`.
+- `-sdkPath` takes the address of the root directory of sdk repo. e.g. /home/azure-sdk-for-net
+- `cadlRelativeFolder` takes the relative path of the cadl project folder in spec repo. e.g. specification/cognitiveservices/AnomalyDetector
+- `-commit` takes the commit of the github hash, e.g. ac8e06a2ed0fc1c54663c98f12c8a073f8026b90
+- `-repo` takes the `<owner>/<repo>` of the spec repo. e.g. Azure/azure-rest-api-specs
+- `-specRoot` takes the file system path of the spec repo. e.g. /home/azure-rest-api-specs
+- You need to provide one of (`-commit`, `-repo`) pair to refer to an URL path of the cadl project and `-specRoot` parameters. If you provide both, `-specRoot` will be ignored.
 
-Run `dotnet build` under project folder ``<sdkPath>/sdk/<service>/<namespace>` to build the project to create the starter package binary.
+When you run the `eng\scripts\automation\Invoke-CadlDataPlaneGenerateSDKPackage.ps1` script, it will:
 
-#### Export the library's public API
+- Create a project folder, install template files from `sdk/template/Azure.Template`, and create `.csproj` and `.sln` files for your new library.
 
-You will need to run the `eng\scripts\Export-API.ps1` script to update the public API listing. This will generate a file in the library's directory `<sdkPath>/sdk/<service>/<namespace>/api` similar to the example found in [sdk\template\Azure.Template\api\Azure.Template.netstandard2.0.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/api/Azure.Template.netstandard2.0.cs).
+    These files are created following the guidance for the [Azure SDK Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md).
 
-e.g. Running the script for a project in `sdk\deviceupdate` would look like this:
+- Generate the library source code files to the directory `<sdkPath>/sdk/<service>/<namespace>/src/Generated`
+- Build the library project to create the starter package binary.
+- Export the library's public API to the directory `<sdkPath>/sdk/<service>/<namespace>/api`
 
-```powershell
-eng\scripts\Export-API.ps1 deviceupdate
-```
```

Comment:
Please remove the reference to the Anomaly Detector swagger file on L226.  That file no longer exists and is causing a link verification failure during nightly runs.   

Please update the section to refer to the Autorest documentation, as done in #33047.

Reply:
updated. Thanks